packages/ia-styles/web-test-runner.config.mjs

```diff
@@ -4,7 +4,7 @@ const filteredLogs = ['Running in dev mode', 'lit-html is in dev mode'];
 
 export default /** @type {import("@web/test-runner").TestRunnerConfig} */ ({
   /** Test files to run */
-  files: 'dist/test/**/*.test.js',
+  // files: 'dist/test/**/*.test.js',
```

Comment:
QUESTION: Is this a temporary commenting out?

Reply:
yes this is, the package is currently styles and does not have any tests. i'd like to keep the line in so when we do add tests, it will be easy to re-enable

Reply:
Could you add a TODO to that effect so we know why that was disabled and when to re-enable it?